tests/drivers/flash/testcase.yaml

```diff
@@ -0,0 +1,9 @@
+tests:
+  drivers.bt_ll_softdevice.flash:
+    platform_allow: nrf52840dk_nrf52840
```

Comment:
Do you also intend to run this test case on the nrf53 network core?

Reply:
Yes, in future. As suggested, commit nRF52840 test scenario first.